### PR TITLE
Fix value conversion and plot with linespoints

### PR DESCRIPTION
**Pull request by ljishen**
- Just a small fix for the `suffixmap`.
- Plot with `linespoints` to make the backing data of the plot clearer.

### CODE REVIEW

File: pp-scripts/pprocess.py [hunk 1]
```diff
@@ -29,16 +29,18 @@
...
     'n' : 1e-9   ,
     'u' : 1e-6   ,
     'm' : 1e-3   ,
     ' ' : 1      ,
     'k' : 1e3    ,
-    'K' : 1e3    ,
-    'M' : 1e6    ,
-    'G' : 1e9    ,
-    'T' : 1e12
+    'K' : 2**3    ,
```
Comment (sbates130272) [inline pp-scripts/pprocess.py]: @ljishen why is this not 2**10??

File: pp-scripts/pprocess.py [hunk 2]
```diff
@@ -352,7 +354,7 @@ def threads(options, args):
     dtLabels=dict()
     dtLabels['title']  = "Threads vs Bandwidth"
     dtLabels['xlabel'] = "FIO threads"
-    dtLabels['ylabel'] = "Bandwidth"
+    dtLabels['ylabel'] = "Bandwidth (bytes/s)"
```
Comment (sbates130272) [inline pp-scripts/pprocess.py]: NIT: I prefer Bytes/s not bytes/s.

**Review by sbates130272 — CHANGES_REQUESTED** [review]
@ljishen thanks for this. A couple of comments.

File: pp-scripts/pprocess.py (hunk 1 above)
Comment (ljishen) [reply pp-scripts/pprocess.py]: How could it happen?! Please bear with me on this. Will fix it.

File: pp-scripts/pprocess.py (hunk 2 above)
Comment (ljishen) [reply pp-scripts/pprocess.py]: OK.

**Review by sbates130272 — APPROVED** [review]
Great, thanks for fixing. I'll approve this and apply the PR!

**Pull request merged**